testing/btest/spicy/tcp-eod-behavior-on-destroy.zeek

```diff
@@ -0,0 +1,39 @@
+# @TEST-REQUIRES: have-spicy
+#
+# @TEST-EXEC: spicyz -d foo.spicy foo.evt -o foo.hlto
+# @TEST-EXEC: zeek -Cr ${TRACES}/http/206_example_b.pcap foo.hlto "Spicy::enable_print=T" >output 2>&1
+# @TEST-EXEC: btest-diff output
+#
+# @TEST-DOC: Exercise &eod behavior when processing is aborted without a regular connection shutdown; regression test for Zeek #4501.
+#
+# @TEST-START-FILE foo.spicy
+module foo;
+
+import zeek;
+
+public type X = unit {
+    : bytes &eod;
+
+    on %init() {
+        print "INIT", zeek::conn_id(), zeek::is_orig();
+    }
+
+    on %done() {
+        print "DONE", zeek::conn_id(), zeek::is_orig(); # should not be called for instance not reaching &eod
```

Comment:
Ups, didn't realize this PCAP had an incomplete flow when I picked it for the reproducer, I guess that is a useful feature to test, even though not every `INIT` having a `DONE` is pretty subtle given the number of frames in that file. Nothing to do.